test/extended/etcd/etcd_storage_path.go

```diff
@@ -246,39 +244,34 @@ func testEtcd3StoragePath(t g.GinkgoTInterface, kubeConfig *restclient.Config, e
 		for _, crd := range etcddataCRDs {
 			errs = append(errs, deleteCRD(ctx, crd.Name, delOptions))
 		}
-		if err := errors.NewAggregate(errs); err != nil {
-			t.Fatal(err)
-		}
+		err := errors.NewAggregate(errs)
+		o.Expect(err).NotTo(o.HaveOccurred())
 	}()
 
-	crds := getCRDs(t, crdClient)
+	crds := getCRDs(crdClient)
 
 	mapper := restmapper.NewDeferredDiscoveryRESTMapper(discocache.NewMemCacheClient(kubeClient.Discovery()))
 	mapper.Reset()
 
 	client := &allClient{dynamicClient: dynamic.NewForConfigOrDie(kubeConfig)}
 
-	if _, err := kubeClient.CoreV1().Namespaces().Create(context.Background(), &kapiv1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: testNamespace}}, metav1.CreateOptions{}); err != nil {
-		t.Fatalf("error creating test namespace: %#v", err)
-	}
+	_, err := kubeClient.CoreV1().Namespaces().Create(context.Background(), &kapiv1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: testNamespace}}, metav1.CreateOptions{})
+	o.Expect(err).NotTo(o.HaveOccurred())
+
 	defer func() {
-		if err := kubeClient.CoreV1().Namespaces().Delete(context.Background(), testNamespace, metav1.DeleteOptions{}); err != nil {
-			t.Fatalf("error deleting test namespace: %#v", err)
-		}
+		kubeClient.CoreV1().Namespaces().Delete(context.Background(), testNamespace, metav1.DeleteOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
We spend far too much time on this topic. We ignore an err, but leave the err check behind. Why not just fix this and be done with it? Otherwise, the next one will readd the `err := ...` because it really looks like a mistake now.